bingo/local_optimizers/continuous_local_opt.py

```diff
@@ -143,6 +146,28 @@ def eval_count(self):
     def eval_count(self, value):
         self._fitness_function.eval_count = value
 
+    @property
+    def param_init_bounds(self):
+        """Iterable: bounds that we use to initialize clo params,
+           should be formatted as an iterable [low, high]
+           where low will be included in the initialization range and
+           high will be excluded"""
+        return self._param_init_bounds
+
+    @param_init_bounds.setter
+    def param_init_bounds(self, value):
+        self._param_init_bounds = value
+
+    @property
+    def optimization_options(self):
+        """Dictionary of options for clo
+        e.g. {"tol": 1e-8, "options": {"maxiter": 1000}}"""
+        return self._optimization_options
+
+    @optimization_options.setter
+    def optimization_options(self, value):
+        self._optimization_options.update(value)
```

Comment:
I would lean towards setting this dictionary rather than updating it.  consider if you did:

```clo.optimization_options = {tol:1e-6, options: {some option}}```

then later did

```clo.optimization_options = {tol:1e-6}```

i would expect that the other options to be removed

Reply:
ah yes I didn't consider this case, I'll update it to be set rather than updated